### DISCUSSION TITLE
Bug: cannot add image to an Album if filename includes a comma (UPDATE: Not a bug - files were stacked)

### DISCUSSION THREAD

**Discussion by electrik** [photoprism/photoprism#1999] [Q&A] [answered]
My library contains files with commas in their names (I manually manage my originals folder).

If the filename includes a comma it cannot be added to an album.

Any help or suggestions?

Cheers

**Comment by lastzero** [comment]
Might be fixed already...
- have you tried our latest version and/or development preview? 
- can you provide logs and/or specific examples to quickly reproduce the problem?

**Reply by electrik** [reply]
Hi Michael

I'm using Docker Compose: PhotoPrism® 220121-2b4c8e1f-Linux-x86_64 (on macOS 10.15.7). I'll try the latest version and report back.

Cheers

**Reply by lastzero** [reply]
If you're using macOS, see also https://gist.github.com/JamesChevalier/8448512 for unicode characters like ü, ä, etc

**Reply by electrik** [reply]
OK, I've not got to the bottom of this.

Firstly, I don't think it's comma related. I've tested with filenames that do not include commas with the result that some files are not added to an album.

Attached is a zip containing a folder with 3 images in it. When I place these images in my originals folder and do a complete rescan I see the 3 images in my 'originals'. When I select the folder and try to create an album from it, only TWO images appear in the album.

I'm stumped.

[009.zip](https://github.com/photoprism/photoprism/files/8007481/009.zip)

**Reply by lastzero** [reply]
Maybe the files have been stacked or so? Did you go through the "pictures missing" checklist in our Troubleshooting Guide already? https://docs.photoprism.app/getting-started/troubleshooting/#missing-pictures

**Reply by graciousgrey** [reply]
I just had a look at your filenames. Two of them are stacked because they have sequential names. You can manually unstack those (https://docs.photoprism.app/user-guide/organize/stacks/#unstack-files) and change the settings so that in the future images are not stacked because of sequential names (https://docs.photoprism.app/user-guide/settings/library/#stacks)

**Reply by electrik** [reply]
Hi lastzero & graciousgrey

Yes, I believe that's the problem I've been having.

Thanks for your time helping me with this. I didn't notice the troubleshooting section in the docs - my bad.

Cheers
Matt

**Comment by AgbegbeTech** [comment]
I need help with this stuff for a project. I'm learning as fast as I can
and I can't compete with dev teams

**Comment by electrik** [comment, marked as answer]
Working as expected. Not a comma issue, my filenames were detected as stacked.
